### PR TITLE
bench-tps retry airdrop and fix subtract panic

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Airdrop transaction can get lost and somehow total_tx_send_count can be lower than max_tx_count.

#### Summary of Changes

See title.

Fixes #

### CODE REVIEW

**Comment by CriesofCarrots** [comment]
Fixes #3227?

**Comment by sakridge** [comment]
@CriesofCarrots that seems like a different issue actually.

**Comment by sakridge** [comment]
@CriesofCarrots or maybe the same issue, but this won't fix it because it just fixes this one print and the assert is before this.

File: bench-tps/src/bench.rs [hunk 1]
```diff
@@ -713,13 +721,19 @@ fn compute_and_report_stats(
         );
     }
 
+    let total_tx_send_count = total_tx_send_count as u64;
+    let num_dropped = if total_tx_send_count > max_tx_count {
+        total_tx_send_count - max_tx_count
```
Comment (pgarg66) [inline bench-tps/src/bench.rs]: We can do ```total_tx_send_count.saturated_sub(max_tx_count)```

Comment (sakridge) [reply bench-tps/src/bench.rs]: good call. I'm not sure I want to merge this though if it hides a bug in the fullnode code.

**Comment by sakridge** [comment]
I'm not sure this is needed.

**Pull request closed without merging**